releasenotes/notes/new_pulse_subclass-44da774612699312.yaml

```
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Introduced a new subclass :class:`~qiskit.pulse.library.ScalableSymbolicPulse`, as a
+    sub class of :class:`~qiskit.pulse.library.SymbolicPulse`. The new subclass behaves
+    the same as :class:`~qiskit.pulse.library.SymbolicPulse`,
+    except that it assumes that the envelope of the pulse includes a complex amplitude
+    pre-factor of the form `amp * exp(1j * angle)`.
+    This envelope shape matches many common pulses, including all of the pulses in
+    the Qiskit Pulse library (which were also converted to 'amp','angle' representation in
+    this release).
+
+    The new subclass removes the non-unique nature of the `amp`,`angle` representation,
+    and correctly compares pulses according to their complex amplitude.
+other:
+  - |
+    QPY version was bumped to 6. Following the introduction of :class:`~qiskit.pulse.library.ScalableSymbolicPulse`
+    the header of saved :class:`~qiskit.pulse.library.SymbolicPulse` objects was modified to include
+    a `class_name` attribute, and the QPY version was bumped to reflect this.
```

```suggestion
other:
  - |
    QPY version was bumped to 6. Following the introduction of :class:`~qiskit.pulse.library.ScalableSymbolicPulse`
    the header of saved :class:`~qiskit.pulse.library.SymbolicPulse` objects was modified to include
    a `class_name_size` attribute, and the QPY version was bumped to reflect this.
```

Please move this to upgrade section. This is an example text for version 5 upgrade.
>  The QPY version format version emitted by :func:`.qpy.dump` has been
increased to version 5. This new format version is incompatible with the
previous versions and will result in an error when trying to load it with
a deserializer that isn't able to handle QPY version 5. This change was
necessary to fix support for representing controlled gates properly and
representing non-default control states.